docs/selectors.md

```diff
@@ -0,0 +1,16 @@
+# Selectors
+
+## Label Selectors in Kubernetes for test run selection
+
+Sometimes, we want to run a particular test only against certain subsets of pods in the Kubernetes cluster (as we may be running multiple simultaneous tests together).
+
+For this purpose, one can make use of the `selector` attribute under a test definition. This follows the kubernetes format of `key=value`, where some of the most common are the `run` key, which indicates a grouping of pods that are part of a particular rungroup.
```

Comment:
A good example of how the above suggestion will improve readability: by spending some time explaining the kubernetes config format before diving into the specifics of the selector section, you will not need to make a digression explaining the general format and its common keys in the middle of explaining the selector attribute.